ext/standard/uniqid.c

```diff
@@ -53,6 +87,24 @@ PHP_FUNCTION(uniqid)
 		Z_PARAM_BOOL(more_entropy)
 	ZEND_PARSE_PARAMETERS_END();
 
+#ifdef __NetBSD__
+	struct uuid uuid;
```

Comment:
I am a bit confused here. uuidgen is on FreeBSD/NetBSD (other oses have different apis) which you detect at configure time. Why the particular NetBSD code path is needed ? we should fall into the code above I think ?

Reply:
Again, I do not really get the code duplication. But not sure it s worth the efforts to be honest considering later comments.

Reply:
How can this be even executed - it's in `elif` section. Basically:

```
#ifdef HAVE_UUIDGEN
...
#elif HAVE_GETTIMEOFDAY
...
#ifdef HAVE_UUIDGEN
```
That doesn't make any sense to me. Am I missing anything?

Reply:
Sorry, I committed it a bit too early, I should I  removed that section that was used in earlier tests. It is fixed now.